django/apps/registry.py

```diff
@@ -255,6 +255,17 @@ def get_registered_model(self, app_label, model_name):
                 "Model '%s.%s' not registered." % (app_label, model_name))
         return model
 
+    @lru_cache.lru_cache(maxsize=None)
+    def get_swappable_setting(self, to_string):
+        # See if anything swapped/swappable matches
+        for model in self.get_models(include_swapped=True):
+            if model._meta.swapped:
+                if model._meta.swapped == to_string:
+                    return model._meta.swappable
+            if ("%s.%s" % (model._meta.app_label, model._meta.object_name)) == to_string and model._meta.swappable:
+                return model._meta.swappable
+        return None
+
```

Comment:
Ah, I saw later that you were just moving this code. Well, it's a good opportunity to understand and document what it does. I insist on keeping `django.apps` at a high level of quality because it's a very critical module.